pkgs/development/libraries/openssl/default.nix

```diff
@@ -1,5 +1,6 @@
 { stdenv, fetchurl, perl
-, withCryptodev ? false, cryptodevHeaders }:
+, withCryptodev ? false, cryptodevHeaders
+, defaultCertificate ? "/etc/ssl/certs/ca-certificates.crt" }:
```

Comment:
This needs to come from the platform stdenv because they all have different locations. Darwin is a bit tricky because it has multiple folders in the keychain format, we might have to recommend to run `certsync` although their distributed curl doesn't seem to have any trouble finding the certs.